pkg/dispatchcli/cmd/install.go

```diff
@@ -18,54 +18,89 @@ import (
 	"strings"
```

Comment:
This file seems unfinished (a lot of commented code, removed libraries that are actually needed, etc.).

Reply:
it is actually, I would also want to have a default config yaml and have the merge function back, WIP on this.